src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Link.cs

```diff
@@ -263,13 +270,13 @@ public async override System.Threading.Tasks.Task RunTaskAsync ()
 				}
 			}
 
-			if (!string.IsNullOrWhiteSpace (AssetsDirectory)) {
-				var assetDir = AssetsDirectory.TrimEnd ('\\');
-				if (!Path.IsPathRooted (assetDir))
-					assetDir = Path.Combine (WorkingDirectory, assetDir);
-				if (!string.IsNullOrWhiteSpace (assetDir) && Directory.Exists (assetDir)) {
+			if (!string.IsNullOrEmpty (AssetsDirectory)) {
+				var assetDir = GetFullPath (AssetsDirectory.TrimEnd ('\\'));
+				if (Directory.Exists (assetDir)) {
 					cmd.Add ("-A");
-					cmd.Add (GetFullPath (assetDir));
+					cmd.Add (assetDir);
+				} else {
+					LogDebugMessage ("asset directory did not exist: " + assetDir);
```

Comment:
can we use `$"asset directory did not exist: {assetDir}"` ? We tend to use that in most places these days.

Reply:
Same for the other log messages.